### PR TITLE
doc pass 12 modules

### PR DESCRIPTION
**Pull request by alganthe**
modules list updated for 3.3.2.

### CODE REVIEW

File: wiki/missionmaker/modules.md [hunk 1]
```diff
@@ -211,12 +203,24 @@ This module allows you to tweak the settings for player names tags.
...
-Show name tags for AI? | Option | Show the name and rank tags for friendly AI units, or by default allows players to choose it on their own. | "Do Not Force"
-Show crew info? | Option | Show vehicle crew info, or by default allows players to choose it on their own. | "Do Not Force"
+Show name tags for AI? | Option | Show the name and rank tags for friendly AI units, or by default allows players to choose it on their own. | Do Not Force
+Show crew info? | Option | Show vehicle crew info, or by default allows players to choose it on their own. | Do Not Force
 Show for Vehicles? | Boolean | Show cursor NameTag for vehicle commander (only if client has name tags enabled). | No
 
+### 1.16 Pointing settings
+*Part of ace_finger*
+
+This module allow you to tweak settings for finger pointing such as is if it's enabled or the distance people can see you point things
```
Comment (bux578) [inline wiki/missionmaker/modules.md]: "This module allows you ..."
"... as if it's ..."
"... the distance at which people can see you pointing at something."

File: wiki/missionmaker/modules.md [hunk 2]
```diff
@@ -363,32 +380,91 @@ Wind Sounds | Boolean | Play wind sounds when Zeus remote controls a unit. | No
...
+
+**Settings:**
+
+Name | Type | Description | Default value
+---- | ---- | ---- | ---- |
+List | List | List of vehicles that will be classified as repair vehicles, separated by commas. |
+Is repair vehicle | Boolean | Is the vehicle classified as a repair vehicle? | Yes
+
+### 2.5 Cargo Settings
+*part of ace_cargo*
```
Comment (bux578) [inline wiki/missionmaker/modules.md]: repair (2.4), cargo (2.5), repair (2.6)

File: wiki/missionmaker/modules.md [hunk 3]
```diff
@@ -396,19 +472,55 @@ This module allows you to change the default Advanced Medical Settings, when [2.
...
+Enabled for | Option | Select what units the advanced medical system will be enabled for. | Players only
 Enable Advanced wounds | Boolean | Allow reopening of bandaged wounds? | No
 Vehicle Crashes | Boolean | Do units take damage from a vehicle crash? | Yes
-Allow PAK | Option | Who can use the PAK for full heal? | "Medics only"
+Allow PAK | Option | Who can use the PAK for full heal? | Medics only
 Remove PAK on use | Boolean | Should PAK be removed on usage? | Yes
-Locations PAK | Option | Where can the personal aid kit be used? | "Vehicles & facility"
-Allow Surgical kit | Option | Who can use the surgical kit? | "Medics only"
+Condition PAK | Option | When can the Personal Aid Kit be used | Stable
+Locations PAK | Option | Where can the personal aid kit be used? | Vehicles & facility
```
Comment (bux578) [inline wiki/missionmaker/modules.md]: Personal Aid Kit or personal aid kit

File: wiki/missionmaker/modules.md [hunk 4]
```diff
@@ -396,19 +472,55 @@ This module allows you to change the default Advanced Medical Settings, when [2.
...
+Allow Surgical kit | Option | Who can use the surgical kit? | Medics only
 Remove Surgical kit | Boolean | Should Surgical kit be removed on usage? | Yes
-Locations Surgical kit | Option | Where can the Surgical kit be used? | "Vehicles & facility"
-Bloodstains | Boolean | Bandaging removes bloodstains. | No
+Locations Surgical kit | Option | Where can the Surgical kit be used? | Vehicles & facility
```
Comment (bux578) [inline wiki/missionmaker/modules.md]: "Surgical Kit" or "surgical kit"?

File: wiki/missionmaker/modules.md [hunk 5]
```diff
@@ -396,19 +472,55 @@ This module allows you to change the default Advanced Medical Settings, when [2.
...
+Condition Surgical Kit | Option | When can the Surgical kit be used? | Anytime
+Heal hitpoints | Boolean | Heal fully bandaged hitpoints. | No
 Pain suppression | Boolean | Pain is only temporarily suppressed not removed. | Yes
 
-### 2.3 Revive Settings
+### 3.2 Medical Menu
+
+This module is used to limit the use of the medical menu.
```
Comment (bux578) [inline wiki/missionmaker/modules.md]: "Medical Menu" or "medical menu"?

File: wiki/missionmaker/modules.md (hunk 3 above)
Comment (alganthe) [inline wiki/missionmaker/modules.md]: It was taken directly from the module.

File: wiki/missionmaker/modules.md (hunk 4 above)
Comment (alganthe) [inline wiki/missionmaker/modules.md]: It was taken directly from the module.

Comment (bux578) [inline wiki/missionmaker/modules.md]: Yeah, I've guessed that :)
inconsistent devs ... ts ts ts

File: wiki/missionmaker/modules.md (hunk 2 above)
Comment (alganthe) [inline wiki/missionmaker/modules.md]: I followed the order in which they appear under ACE logitistics.

Comment (bux578) [inline wiki/missionmaker/modules.md]: ok. It's not terrible bad

File: wiki/missionmaker/modules.md (hunk 5 above)
Comment (alganthe) [inline wiki/missionmaker/modules.md]: The header is like that for normalization across the file.

File: wiki/missionmaker/modules.md (hunk 4 above)
Comment (alganthe) [inline wiki/missionmaker/modules.md]: I'll fix this.
I'll fix the module too and PR it.

Comment (bux578) [inline wiki/missionmaker/modules.md]: You're awesome. Thanks!

**Pull request merged**